authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationHDFSPlugin.java

```diff
@@ -679,4 +680,57 @@ public Boolean onMetadataUpdated(MetadataObjectChange... changes) throws Runtime
     }
     return Boolean.TRUE;
   }
+
+  @Override
+  protected String getServiceType() {
+    return "hdfs";
```

Comment:
better not use hardcoded string here?

Reply:
I will put these constants to one place to manage them